CMake/ctkMacroGenerateMocs.cmake

```diff
@@ -28,8 +30,10 @@ function(QT4_GENERATE_MOCS)
     if(CTK_QT_VERSION VERSION_GREATER "4")
       if(Qt5_VERSION VERSION_LESS "5.6")
         QT5_CREATE_MOC_COMMAND(${abs_file} ${moc_file} "${_moc_flags}" "" "")
-      else()
+      elseif(Qt5_VERSION VERSION_LESS "5.15.0")
         QT5_CREATE_MOC_COMMAND(${abs_file} ${moc_file} "${_moc_flags}" "" "" "")
+      else()
+        qt5_generate_moc(${abs_file} ${moc_file})
```

Comment:
`qt5_generate_moc` internally calls `QT5_GET_MOC_FLAGS` and ensured no warnings are reported.

Reply:
Add this as a comment to the source code.